src/FreeAge/server/main.cpp

```diff
@@ -97,7 +97,7 @@ int main(int argc, char** argv) {
     if (player->state == PlayerInMatch::State::Joined) {
       int index = playersInGame.size();
       int colorIndex = player->playerColorIndex;
-      std::shared_ptr<PlayerInGame> newPlayer(new PlayerInGame(index, colorIndex, gameData));
+      std::shared_ptr<PlayerInGame> newPlayer(new PlayerInGame(index, colorIndex, gameData, Civilization::Byzantines));
```

Comment:
I will leave propagating the civilization selection up to the lobby gui for someone else...